polling_stations/apps/file_uploads/templates/file_uploads/upload.html

```diff
@@ -27,61 +27,70 @@ <h2>Javascript disabled</h2>
     </noscript>
 
     <h1>{{ council.name }}</h1>
-    <h2>Upload EMS export</h2>
-    <p>
-        Use the buttons below to select and upload a polling station export file from your computer.
-        Instructions on how to create the export can be found <a href="https://democracyclub.org.uk/projects/polling-stations/upload/">here</a>.
-    </p>
-    <p>
-        In the event that some of the stations in your council area are managed by neighbouring councils for the GE,
-        please ensure that they are aware that they need to provide this data to us.
-    </p>
-
-    <form id="file_upload_form" method="POST">
-        {% if SHOW_DATE_PICKER %}
+    {%if NO_UPCOMING_ELECTIONS %}
```

Comment:
minor style point
```suggestion
    {% if NO_UPCOMING_ELECTIONS %}
```

Reply:
fixed in d630b7ebef6677f53d5a1fcf9ef1e0c2583e2f14